src/Microsoft.AspNet.Mvc.Xml/XmlSerializerInputFormatter.cs

```diff
@@ -96,7 +123,7 @@ public bool CanRead(InputFormatterContext context)
         /// <returns>The type to which the XML will be deserialized.</returns>
         protected virtual Type GetSerializableType([NotNull] Type declaredType)
```

Comment:
This was a new method added by Tony in his latest commit...i need to handle it as i do in the output formatters...will update this PR